### PR TITLE
End points getById and getAll done

### PR DESCRIPTION
**Pull request by Hulyaoren**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -8,10 +8,10 @@ server.error.include-message=always
 server.error.include-stacktrace= never
 ###########################################################
 #database credentials
-spring.datasource.url= jdbc:postgresql://localhost:5432/school_management_b262_instructor_db
-spring.datasource.username= postgres
+spring.datasource.url= jdbc:postgresql://localhost:5432/school_management_b262_db
+spring.datasource.username= techprodt
```
Comment (sametyaprak) [inline src/main/resources/application.properties:12]: you should not change anything in this file

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -8,10 +8,10 @@ server.error.include-message=always
...
 #postgres
-spring.datasource.password= changeme
+spring.datasource.password= password
```
Comment (sametyaprak) [inline src/main/resources/application.properties:14]: Also for this

File: src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java [hunk 3]
```diff
@@ -47,17 +47,17 @@ public class EducationTermController {
   @PreAuthorize("hasAnyAuthority('Admin','Dean','ViceDean','Teacher')")
   @GetMapping("/getAll")
   public List<EducationTermResponse> getAllEducationTerms(){
-    //return educationTermService.getAllEducationTerms();
-    return null;
+    return educationTermService.getAllEducationTerms();
+    //return null;
```
Comment (sametyaprak) [inline src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java:51]: please remove //return null;

File: src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java [hunk 4]
```diff
@@ -47,17 +47,17 @@ public class EducationTermController {
...
   
 
   //TODO HULYA
   @PreAuthorize("hasAnyAuthority('Admin','Dean','ViceDean','Teacher')")
   @GetMapping("/{id}")
   public EducationTermResponse getEducationTermById(@PathVariable Long id){
-    //return educationTermService.findById(id);
-    return null;
+    return educationTermService.findById(id);
+    //return null;
```
Comment (sametyaprak) [inline src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java:60]: please remove //return null;

File: src/main/resources/application.properties (hunk 1 above)
Comment (Hulyaoren) [reply src/main/resources/application.properties]: username is changed back to postgres

File: src/main/resources/application.properties (hunk 2 above)
Comment (Hulyaoren) [reply src/main/resources/application.properties]: password is corrected

File: src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java (hunk 3 above)
Comment (Hulyaoren) [reply src/main/java/com/project/schoolmanagment/controller/businnes/EducationTermController.java]: comments are removed

**Review by sametyaprak — APPROVED** [review]

**Pull request merged**